docs/source/technical_tutorials/visualization/visualizing_titiler-pgstac.ipynb

```diff
@@ -0,0 +1,751 @@
+{
```

Comment:
<p>Interesting, why do we use the titiler endpoint to get STAC info instead of directly gettting it from STAC?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/423/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I was wondering the same thing when I first read this. Your reason makes sense @hrodman. A consideration I just had was what if the user intends to use a titiler instance that does not include the renders information. (Of course not all stac records will work with all titiler instances, but some will). But I can see how considering that use case may be outside of the scope of this notebook.</p>